modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -56,6 +56,19 @@ def create_session?
     end
   end
 
+  def run
+    results = super
+    logins = results.flat_map { |_k, v| v[:successful_logins] }
+    sessions = results.flat_map { |_k, v| v[:successful_sessions] }
+    print_status("Bruteforce completed, #{logins.size} credentials were successful.")
+    if datastore['CreateSession']
+      print_status("#{sessions.size} MSSQL sessions were opened successfully.")
+    else
+      print_status('You can open an MSSQL session with these credentials and CreateSession set to true')
```

Comment:
```suggestion
      print_status('You can open an MSSQL session with these credentials and %grnCreateSession%clr set to true')
```

(And in the other places)